src/System.Text.Json/src/System/Text/Json/Serialization/Converters/JsonValueConverterJsonElement.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Json.Serialization.Policies;
+
+namespace System.Text.Json.Serialization.Converters
+{
+    internal sealed class JsonValueConverterJsonElement : JsonValueConverter<JsonElement>
+    {
+        public override bool TryRead(Type valueType, ref Utf8JsonReader reader, out JsonElement value)
+        {
+            if (JsonDocument.TryParseValue(ref reader, out JsonDocument document))
+            {
+                value = document.RootElement.Clone();
+                document.Dispose();
+                return true;
+            }
+            else
+            {
+                value = default;
+                return false;
+            }
+        }
+
+        public override void Write(JsonElement value, Utf8JsonWriter writer)
+        {
+            value.WriteAsValue(writer);
+        }
+
+        public override void Write(Span<byte> escapedPropertyName, JsonElement value, Utf8JsonWriter writer)
```

Comment:
@steveharter, this should be `ReadOnlySpan<byte>`. Is there a reason it needs to be writable?

https://github.com/dotnet/corefx/blob/648d0724c2b9224cace4b1e179fc20904fe27aa4/src/System.Text.Json/src/System/Text/Json/Serialization/Policies/JsonValueConverter.cs#L11